team-cards/src/Components/Search.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import "../Styles/Search.css";
+
+const searchTeamMembers = (e, searchTerm, setFilteredData, data) => {
+  e.preventDefault();
+  const filteredData = data.filter((teamMember) =>
+    teamMember.name.toLocaleLowerCase().includes(searchTerm.toLocaleLowerCase())
+  );
+  setFilteredData(filteredData);
+};
+
+const Search = ({ setFilteredData, data }) => {
+  const [searchTerm, setSearchTerm] = useState();
```

Comment:
What gets set in useState when we don't pass a default value.

Reply:
It will be undefined.